cmds/core/cp/cp_test.go

```diff
@@ -260,8 +261,103 @@ func TestCpSymlink(t *testing.T) {
 		if err := cpArgs([]string{newName, dst}); err != nil {
 			t.Fatalf("cp(%q -> %q) = %v, want nil", newName, dst, err)
 		}
-		if err := cmp.IsEqualTree(cp.Default, newName, dst); err != nil {
+		if err := IsEqualTree(cp.Default, newName, dst); err != nil {
 			t.Fatalf("The copy %q -> %q failed: %v", newName, dst, err)
 		}
 	})
 }
+
+// isEqualFile compare two files by checksum
+func isEqualFile(fpath1, fpath2 string) error {
```

Comment:
something I am not sure of: why move this out of the package and into the command? What am I missing?

Reply:
```
// Copyright 2018 the u-root Authors. All rights reserved
// Use of this source code is governed by a BSD-style
// license that can be found in the LICENSE file.

// Package cmp compares trees of files.
//
// cmp is an internal package for pkg/cp's and cmds/core/cp's tests.
```

pkg/cp/cmp provides functions only for testing of cmds/core/cp and the old implementation of tests for pkg/cp. 
My idea was to get rid of it completly in cp for the sake of consistency. The functionality, if required, should not be hidden in a different package but be part of the test file itself. 

The reason I moved it to cmds/core/cp is, that I dont want to touch the functionality of cmds/core/cp in this PR. This is a problem for future Chris. This gives me a plain field to implement the tests for pkg/cp.

Also I would consider it bad practice to have a subpackage which only provide some functions for testing, because then we write tests for test functions and i think that is not what we wanna do here.